CHANGELOG.md

```diff
@@ -2,13 +2,15 @@
 
 ### features
 
-* `[jest-mock]` Add util methods to create async functions. 
-  ([#5318](https://github.com/facebook/jest/pull/5318)) 
+* `[jest-mock]` Add util methods to create async functions.
+  ([#5318](https://github.com/facebook/jest/pull/5318))
 
 ### Fixes
 
 * `[jest]` Add `import-local` to `jest` package.
   ([#5353](https://github.com/facebook/jest/pull/5353))
+* `[expect]` Support class instances in `.toHaveProperty()` matcher.
```

Comment:
This also affected `.toMatchObject()` as far as I can tell.

Reply:
in a good or bad way?

Reply:
In a very good way. It basically enables `.toMatchObject()` to check if a given class implements an interface.

But in my opinion this should be included in the changelog regardless of whether one finds the change useful or not.

Reply:
PR welcome 🙂

Reply:
Should I include some tests just to make sure I identified that behavior correctly?

Reply:
That would be awesome!